docs/source/models/field_ionization.rst

```diff
@@ -0,0 +1,166 @@
+    | Multiphoton         | None, yet                   |                           |
+    +---------------------+-----------------------------+---------------------------+
+    | Tunneling           | * ``Keldysh``               | * [BauerMulser1999]_      |
+    |                     | * ``ADKLinPol``             | * [DeloneKrainov]_        |
+    |                     | * ``ADKCircPol``            | * [DeloneKrainov]_        |
+    +---------------------+-----------------------------+---------------------------+
+    | Barrier Suppression | * ``BSI``                   | * [MulserBauer2010]_      |
+    |                     | * ``BSIEffectiveZ``         | * [ClementiRaimondi1963]_ |
+    |                     |                             |   [ClementiRaimondi1967]_ |
+    |                     | * ``BSIStarkShifted`` (R&D) | * [BauerMulser1999]_      |
+    +---------------------+-----------------------------+---------------------------+
+
+.. attention::
+
+    Models marked with "(R&D)" are under *research and development* and should be used with care.
+
+Usage
+-----
+
+Input for ionization models is defined in :ref:`speciesDefinition.param <usage-params-core>`, :ref:`ionizer.param and ionizationEnergies.param <usage-params-extensions>`.
+
+
+Barrier Suppression Ionization
+------------------------------
+
+The so-called barrier-suppression ionization regime is reached for strong fields where the potential barrier binding an electron is completely suppressed.
+
+Tunneling Ionization
+--------------------
+
+Tunneling ionization describes the process during which an initially bound electron quantum-mechanically tunnels through a potential barrier of finite height.
+
+Keldysh
+"""""""
+
+.. math::
+
+    \Gamma_\mathrm{K} = \frac{\left(6 \pi\right)^{1/2}}{2^{5/4}} E_\mathrm{ip} \left( \frac{F}{(2 E_\mathrm{ip})^{3/2}} \right)^{1/2} \exp\left(-\frac{2 \left(2 E_\mathrm{ip}\right)^{3/2}}{3 F}\right)
+
+According to the equation (9) in [BauerMulser1999]_ the Keldysh ionization rate has been implemented. See also [Keldysh]_ for the original work.
+
+.. note::
+
+    Assumptions:
+
+        * low field - perturbation theory
+        * :math:`\omega_\mathrm{laser} \ll E_\mathrm{ip}`
+        * :math:`F \ll F_\mathrm{BSI}`
+        * tunneling is instantaneous
+
+
+Ammosov-Delone-Krainov (ADK)
+""""""""""""""""""""""""""""
+
+.. math::
+
+    \begin{align}
+        \Gamma_\mathrm{ADK} &= \underbrace{\sqrt{\frac{3 n^{*3} F}{\pi Z^3}}}_\text{lin. pol.} \frac{F D^2}{8 \pi Z} \exp\left(-\frac{2Z}{3n^{*3}F}\right) \\
+        D &\equiv \left( \frac{4 \exp Z^3}{F n^{*4}} \right)^{n^*} \hspace{2cm} n^* = \frac{Z}{\sqrt{2 E_\mathrm{ip}}}
+    \end{align}
+
+We implemented equation (7) from [DeloneKrainov]_ which is a *simplified result assuming s-states* (since we have no atomic structure implemented, yet).
+Leaving out the pre-factor distinguishes ``ADKCircPol`` from ``ADKLinPol``.
```

Comment:
OPTIONAL:
This statement is not clear to me, please specify what you mean with this sentence.